### DISCUSSION TITLE
[Request] Please, add support for Linux ARMv7 & ARM64

### DISCUSSION THREAD

**Discussion by pioj** [TSnake41/raylib-lua#12] [Ideas]
HI, first of all **BIG THANKS** for such a magnificient work. It runs out of the box!

Could you try to provide prebuilt binaries for Linux on ARM also, please?

There are lots of developer communities around ARM devices today, but for many users, having to setup everything is their 1st barrier. And it's difficult enough to disencourage and make them go directly to, let's say, Unity then target only Android instead...

Maybe you can bring a more direct, easier alternative, simple enough to start coding games...

(_take for example the [/r/SBCGaming](https://www.reddit.com/r/SBCGaming/) subreddit..._)

Thanks again.

**Comment by TSnake41** [comment]
That's a good idea, but I don't have the tooling nor the hardware (except for ARMv7) to build and test raylib-lua against.

There is still some work needed to build for those architecture, definetely something to consider.
